app/controllers/state_file/questions/landing_page_controller.rb

```diff
@@ -1,14 +1,36 @@
 module StateFile
   module Questions
     class LandingPageController < QuestionsController
-      include StartIntakeConcern
-      before_action :before_update, only: :update
+      skip_before_action :redirect_if_no_intake
 
-      def before_update
+      def edit
+        @state_name = StateFileBaseIntake::STATE_CODE_AND_NAMES[params[:us_state]]
+        if current_intake.present?
+          if current_intake.primary_first_name.present?
+            @user_name = "#{current_intake.primary_first_name} #{current_intake.primary_middle_initial} #{current_intake.primary_last_name}"
+          elsif current_intake.raw_direct_file_data.present?
+            begin
+              @user_name = current_intake.direct_file_data.node.at("Filer NameLine1Txt").text.gsub("&lt;", " ")
```

Comment:
this will be all caps in a lot of cases, yes? just confirming it's intentional

Reply:
This is a fallback for cases where they have not completed the NameDOB step - hopefully that is a really small subset